.scalafix.test.conf

```diff
@@ -0,0 +1,12 @@
+rules = [
+  Http4sGeneralLinters
+  Http4sUseLiteralsSyntax
+  LeakingImplicitClassVal
+  OrganizeImports
```

Comment:
@danicheg the test config also enforces organize imports, right?

Reply:
Right. Then my hypothesis about Test config is wrong. But why all is good at local 🤯